### PR TITLE
Final hand in week 2

### PR DESCRIPTION
**Pull request by isabellwastfelt**

### CODE REVIEW

**Comment by isabellwastfelt** [comment]
https://inspiring-bhaskara-9a841b.netlify.app/code/

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,275 @@
...
+  text-decoration: none;
+  font-weight: bold;
+  margin-left: 20px;
+}
+
+a:hover {
+  background: gray;
+  transition: .5s;
+  border-radius: 20px;
+}
```
Comment (JensBergqvist) [inline code/style.css:57]: The links are beautiful! I really like the effect you have when you hover over them!

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,275 @@
...
+
+.logo-hidden {
+  display: none;
+}
+
+
+/* ----- Big news ----- */
+.big-news {
+  display: grid;
+  grid-template-columns: 50% 50%;
```
Comment (JensBergqvist) [inline code/style.css:73]: Maybe try just 1 column with 100% so everything is in 1 row for mobile.

**Review by JensBergqvist — COMMENTED** [review]
Great structure in the code overall! I know where I am what does what at all times!
You made a beautiful site and should be really proud of it!

I only saw 1 thing that might need a change In mobile verision. in the "Big news"-section The info about the new magazine should be below the image instead of next to it? like instead of having grid-template-columns: 50% 50%. maybe try with  :100%? I think it was on row 73 in the Css file.

**Pull request closed without merging**